### PR TITLE
Theme showcase: simplify activation modal UX & copy

### PR DESCRIPTION
**Pull request by fushar**
Related to:

- https://github.com/Automattic/wp-calypso/issues/83152

## Proposed Changes

When activating a theme, we now:

1. remove the checkbox (reducing the number of clicks)
2. mention the old (existing) theme name
3. make the copy less scary

|Before|After|
|-|-|
|image|image|

## Testing Instructions

1. Go to `/themes`
2. Activate a theme
3. Make sure the modal shows up as expected above
4. Make sure the theme can actually be activated after confirming the modal

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/themes/activation-modal.jsx [hunk 1]
```diff
@@ -124,13 +107,17 @@ export class ActivationModal extends Component {
...
 					<h1 className="activation-modal__heading">
 						{ translate( 'Activate %(themeName)s', {
-							args: { themeName },
+							args: { themeName: newTheme.name },
 						} ) }
 					</h1>
 					<p className="activation-modal__description">
 						{ translate(
-							'After activation, this layout will replace your existing homepage. But you can still access your old content. {{a}}Learn more{{/a}}.',
+							'You’re about to change your active theme from {{strong}}%(activeThemeName)s{{/strong}} to {{strong}}%(newThemeName)s{{/strong}}.{{br}}{{/br}}{{br}}{{/br}}This will replace your homepage. Don’t worry, your old content can still be accessed. {{a}}Learn more{{/a}}.',
```
Comment (miksansegundo) [inline client/my-sites/themes/activation-modal.jsx]: What do you think about the ideas behind the following copy variations? cc: @michaelpick 

I got inspired by the copy in the section "What Happens to Your Old Content" in [Learn more](https://wordpress.com/support/themes/changing-themes/#what-happens-to-your-old-content). 

### **In this PR: "Don’t worry, your old content can still be accessed."**

Screenshot 2566-10-19 at 13 20 56


### **Suggestion 1: "You can recover it later."**

Screenshot 2566-10-19 at 13 08 06

### **Suggestion 2: "Your old content remains on the site and you can recover it."**

Screenshot 2566-10-19 at 13 10 08

### **Suggestion 3: "If you wish to recover it, reactivate Fewer to view and copy any of that content."**

Screenshot 2566-10-19 at 13 14 24

Comment (okmttdhr) [inline client/my-sites/themes/activation-modal.jsx]: 📝  somewhat related issue: https://github.com/Automattic/wp-calypso/issues/81878

**Review by okmttdhr — APPROVED** [review]
Tested and looked good to me 👍

File: client/my-sites/themes/activation-modal.jsx
Comment (miksansegundo) [inline client/my-sites/themes/activation-modal.jsx:1]: Together with this PR or as a follow-up task, we have to update the screenshot of the modal that lives in the section [How to Switch Themes](https://wordpress.com/support/themes/changing-themes/#how-to-switch-themes).

What do you think about update the link "Learn more" to go directly to the section "What Happens to Your Old Content" https://wordpress.com/support/themes/changing-themes/#what-happens-to-your-old-content

**Review by miksansegundo — APPROVED** [review]

File: client/my-sites/themes/activation-modal.jsx
Comment (fushar) [reply client/my-sites/themes/activation-modal.jsx:1]: Ooohhh yes, this is a good idea!!! ❤️

File: client/my-sites/themes/activation-modal.jsx (hunk 1 above)
Comment (fushar) [reply client/my-sites/themes/activation-modal.jsx]: Thanks for the great suggestions! I went ahead with suggestion 2 with a small tweak:

```
Your old content remains on the site and can be recovered later.
```

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/9408956

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @fushar for including a screenshot in the description! This is really helpful for our translators.

**Comment by michaelpick** [comment]
> What do you think about the ideas behind the following copy variations? cc: @michaelpick

Hi, @miksansegundo! Looks like this might already have shipped, but if not (or for future iteration) 'recover' can feel a little close to fixing something that went wrong without extra context. Francisco and I came up with a few alternatives in case that's useful:

- This will replace your homepage, but your content will remain accessible. Learn more. 
- This will replace your homepage content but it will be saved and ready to reuse any time. Learn more.
- This will replace your homepage, but your previous content can be recovered any time. Learn more.

**Comment by fushar** [comment]
Thanks @michaelpick! It's not too late!

> 'recover' can feel a little close to fixing something that went wrong without extra context.

This makes sense. I went ahead with the first option ("This will replace your homepage, but your content will remain accessible.") 😄

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/9408956).

**Pull request merged**